src/precice/SolverInterface.hpp

```diff
@@ -254,10 +254,9 @@ class SolverInterface {
    * Use isMeshConnectivityRequired() to check if the current participant can make use of the connectivity.
    *
    *
-   * Always set the mesh connectivity of the highest dimensionality available.
-   * preCICE ensures the existence of hierarchical entries for the projection fallback.
-   * Prefer to use bulk versions, as they allows preCICE to efficiently avoid duplicates.
-   * preCICE removes all connectivity duplicates in initialize().
+   * Only set the mesh connectivity that you require.
+   * preCICE removes all connectivity duplicates in initialize() and ensures
+   * the existence of hierarchical entries for the projection fallback.
```

Comment:
I get what you are saying here, but I'm not sure if users will understand what you mean here. The connectivity is anyway somewhat complicated. I would recommend to make this description a bit more verbose, or even remove the last part (does the user need to know about fallbacks etc.?).

Reply:
I also don't understand exactly what you mean by "hierarchical entries".

> Only set the mesh connectivity that you require

Will users understand what we mean with this?

Would it make sense to extend the examples below by what preCICE will remove?

Reply:
Extended the docs and listed `{2D, 3D}x{Surface, Volume}`.